src/pages/Admin/Challenge/ChallengeMembershipPreferencePage.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC, useMemo } from 'react';
+import { PageProps } from '../../common';
+import { useChallenge, useUpdateNavigation } from '../../../hooks';
+import ChallengePreferenceContainer from '../../../containers/preferences/challenge/ChallengePreferenceContainer';
+import ChallengeMembershipPreferenceView from '../../../views/Challenge/ChallengeMembershipPreferenceView';
+
+const ChallengeMembershipPreferencePage: FC<PageProps> = ({ paths }) => {
+  const { hubId, challengeId } = useChallenge();
+  const currentPaths = useMemo(() => [...paths, { value: '', name: 'preferences', real: false }], [paths]);
+  useUpdateNavigation({ currentPaths });
+
+  return (
+    <ChallengePreferenceContainer hubId={hubId} challengeId={challengeId}>
+      {(entities, state, actions) => {
+        const authPrefs = entities.preferences.filter(x => x.definition.group === 'MembershipChallenge');
+        return (
+          <ChallengeMembershipPreferenceView
+            entities={{ preferences: authPrefs }}
+            state={{ loading: state.loading, error: state.error }}
+            actions={{ onUpdate: actions.onUpdate }}
+            options={{}}
```

Comment:
Makes sense to make `options` optional? 🙂

Reply:
Actually what I wanted it to rearrange the params of `ViewProps` and `ContainerProps` so that only **entities** and **state** are mandatory.
Introducing optional parameters right now will introduce type errors the likes of 
` Property 'xxx' does not exist on type 'xxx | undefined'.`